TangoTest.cpp

```diff
@@ -692,6 +700,13 @@ void TangoTest::init_device()
 	*attr_ushort_scalar_read = 0;
 	attr_ushort_scalar_write = 0;
 
+	attr_devenum_scalar_read = new devenum_scalarEnum;
+	*attr_devenum_scalar_read = _SCALAR_E;
+	attr_devenum_scalar_write = _SCALAR_A;
```

Comment:
attr_devenum_scalar_write (devenum_scalar_wEnum type) seems to be used when writing devenum_scalar and devenum_scalar_w attributes.

But devenum_scalar attribute seems to expect devenum_scalar_Enum type.

I think there are some inconsistencies here which could bring confusion.

I also think devenum_scalar_w and devenum_scalar_rww should use the same enum type.